src/js/gi/components/profile/CalculatorForm.jsx

```diff
@@ -16,7 +16,6 @@ class CalculatorForm extends React.Component {
     this.renderScholarships = this.renderScholarships.bind(this);
     this.renderTuitionAssist = this.renderTuitionAssist.bind(this);
     this.renderEnrolled = this.renderEnrolled.bind(this);
```

Comment:
Probably not, but it's just something I do anyway just in case. The one time I forget, I'll probably end up with some confusing bug related to `this`. If anything, it's like a quick table of contents for the functions in the class.